src/main/java/edu/ksu/canvas/attendance/submitter/AssignmentValidator.java

```diff
@@ -66,11 +66,11 @@ public AttendanceAssignment validateCanvasAssignment(AttendanceAssignment assign
      * Returns true if configuration in the form and in the db is the same
      */
     private boolean isAssignmentConfigurationSaved(AttendanceAssignment assignmentConfigurationFromSetup, AttendanceAssignment attendanceAssignmentSaved) {
-        return assignmentConfigurationFromSetup.getAssignmentPoints().doubleValue() == attendanceAssignmentSaved.getAssignmentPoints().doubleValue()
-                && assignmentConfigurationFromSetup.getExcusedPoints().doubleValue() == attendanceAssignmentSaved.getExcusedPoints().doubleValue()
-                && assignmentConfigurationFromSetup.getAbsentPoints().doubleValue() == attendanceAssignmentSaved.getAbsentPoints().doubleValue()
-                && assignmentConfigurationFromSetup.getTardyPoints().doubleValue() == attendanceAssignmentSaved.getTardyPoints().doubleValue()
-                && assignmentConfigurationFromSetup.getPresentPoints().doubleValue() == attendanceAssignmentSaved.getPresentPoints().doubleValue();
+        return Double.parseDouble(assignmentConfigurationFromSetup.getAssignmentPoints()) == Double.parseDouble(attendanceAssignmentSaved.getAssignmentPoints())
```

Comment:
We need to do something safer to compare floating point numbers. 

Consider using
Math.abs(f1 - f2) < epsilon
where epsilon is a very small number like 0.00000001, depending on the desired precision

See -- https://stackoverflow.com/questions/1088216/whats-wrong-with-using-to-compare-floats-in-java